cocos/spine/skeleton.ts

```diff
@@ -707,7 +708,7 @@ export class Skeleton extends Renderable2D {
         return this._cacheMode !== AnimationCacheMode.REALTIME;
     }
 
-    public update (dt: number) {
+    public updateAnimation (dt: number) {
```

Comment:
To be fully comply type check, you can have an interface

```
interface SkeletonAnimation2D {
    updateAnimation (dt: number): void;
}
```

Implement this interface in  both spine skeleton and db armature

Reply:
Same as above. I realize that there is no need to use a unified interface to increase coupling